src/components/Button/Button.stories.tsx

```diff
@@ -7,17 +7,50 @@ import Icon from '../Icon';
 export default {
   title: 'Molecules/Buttons/Button',
   component: Button,
+  args: {
+    children: 'Button',
+    variant: 'primary',
+    status: 'brand',
+    fullWidth: false,
+    size: 'lg',
+    loading: false,
+  },
+  argTypes: {
+    variant: {
+      control: {
+        type: 'select',
+        options: ['primary', 'secondary', 'icon', 'link'],
+      },
+    },
+    status: {
+      control: {
+        type: 'select',
+        options: ['brand', 'neutral', 'success', 'warning', 'error'],
+      },
+    },
+    size: {
+      control: {
+        type: 'select',
+        options: ['sm', 'md', 'lg'],
+      },
+    },
+    fullWidth: {
+      control: 'boolean',
+    },
+    loading: {
+      control: 'boolean',
+    },
+  },
 } as Meta;
 
 type Args = React.ComponentProps<typeof Button>;
 
 const Template: Story<Args> = (args) => <Button {...args} />;
 
 export const Primary = Template.bind({});
-Primary.args = { children: 'Button' };
```

Comment:
I also remove the children props that are the same as the default value (all stories without an icon).